src/som/VM.java

```diff
@@ -86,7 +89,7 @@ public static void setEngine(final PolyglotEngine e) {
 
   private int lastExitCode = 0;
   private volatile boolean shouldExit = false;
-  private final VmOptions options;
+  private final VMOptions options;
```

Comment:
This change looks like it might be the result of having worked with an earlier version. I recently renamed `VmOptions`.